app/src/main/java/de/danoeh/antennapod/fragment/preferences/NotificationPreferencesFragment.java

```diff
@@ -0,0 +1,30 @@
+package de.danoeh.antennapod.fragment.preferences;
+
+import android.os.Bundle;
+import androidx.preference.PreferenceFragmentCompat;
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.activity.PreferenceActivity;
+import de.danoeh.antennapod.core.preferences.GpodnetPreferences;
+
+public class NotificationPreferencesFragment extends PreferenceFragmentCompat {
+
+    private static final String TAG = "NotificationPrefFragment";
+    private static final String PREF_GPODNET_NOTIFICATIONS = "pref_gpodnet_notifications";
+
+    @Override
+    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
+        addPreferencesFromResource(R.xml.preferences_notification);
```

Comment:
Please rename the xml file to `preferences_notifications` (plural). I think that reads a bit better. The Java class name is fine as-is

Reply:
renamed as preferences_notifications